dags/benchmark/pytorchxla_torchbench.py

```diff
@@ -19,21 +19,27 @@
 from configs import vm_resource
 from configs.benchmark.pytorch import pytorchxla_torchbench_config as config
 
+_MODELS = [
+    "BERT_pytorch",
+    "Background_Matting",
+    "hf_Bert",
+    "Super_SloMo",
+    "tts_angular",
+]
 with models.DAG(
     dag_id="pytorch_nightly_torchbench",
     schedule=None,
```

Comment:
Can you also add schedule info like this? You may want to update the time properly to avoid conflict with other dags https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/xlml/pytorchxla_llama.py#L21

Reply:
Added for every Tuesday at 5:00PM.